ambry-store/src/main/java/com/github/ambry/store/BlobStoreStats.java

```diff
@@ -148,14 +150,13 @@ static StatsSnapshot convertStoreUsageToContainerStatsSnapshot(Map<Short, Map<Sh
         TimeUnit.MINUTES.toMillis(config.storeStatsRecentEntryProcessingIntervalInMinutes),
         config.storeStatsWaitTimeoutInSecs, config.storeEnableBucketForLogSegmentReports,
         config.storeCompactionPurgeDeleteTombstone, time, longLiveTaskScheduler, shortLiveTaskScheduler,
-        diskIOScheduler, metrics);
+        diskIOScheduler, metrics, TimeUnit.MINUTES.toMillis(config.storeGetValidSizeIntervalInMinutes));
```

Comment:
Formatting. This line is too long.

Reply:
This line is not exceed the right margin. Do you mean the constructer after this one? I've formatted that one.